### DISCUSSION TITLE
Replacing a single server in a cluster

### DISCUSSION THREAD

**Discussion by Jazz9** [minio/minio#17674] [Q&A] [answered]
We have a dev cluster that I am testing a node replacement on.

I can't see to find a way to figure out when it has 100% rebuilt? Is there a way to view progress?

**Comment by vadmeste** [comment, marked as answer]
> I can't see to find a way to figure out when it has 100% rebuilt? Is there a way to view progress?

`mc admin heal <alias>` will show the healing progress of the disks in that rebuilt node

**Reply by Jazz9** [reply]
Thanks! That is perfect.
